iceoryx_dds/include/iceoryx_dds/dds/data_reader.hpp

```diff
@@ -23,7 +23,7 @@ namespace iox
 {
 namespace dds
 {
-using IdString = iox::cxx::string<100u>;
+using IdString_t = iox::cxx::string<100u>;
```

Comment:
@ithier the same alias is also present `data_writer.hpp` and `dds_types.hpp`. Unfortunately, `dds_types.hpp` includes both, `data_reader.hpp` and `data_writer.hpp`. Any proposal what could be a common place for this alias?

Reply:
Why do we not use the capro::IdString_t?

Reply:
replaced with capro::IdString_t